spec/requests/portfolio_items_spec.rb

```diff
@@ -267,6 +267,7 @@
       it 'patches the record' do
         expect(json["name"]).to eq valid_attributes[:name]
         expect(json["description"]).to eq valid_attributes[:description]
+        expect(json["workflow_ref"]).to eq valid_attributes[:workflow_ref]
```

Comment:
Could reduce the 3 tests to `expect(json).to include(valid_attributes.stringify_keys)`

@syncrou thoughts?

Reply:
@gmcculloug I very much like that.